docs/baremetal_example.md

```diff
@@ -0,0 +1,5 @@
+## Bare metal example
+
+To start using the bare metal profile on Mbed OS 5, please see the blinky bare metal example:
```

Comment:
Isn't it normally Blinky?

Reply:
We have a new example that's basically blinky for bare metal; it's different from the standard blinky.

Reply:
Oh, do you just mean capitalization?

Reply:
Yes, capitalisation.